lib/rails/sharding.rb

```diff
@@ -1,3 +1,5 @@
+require 'dotenv/load'
```

Comment:
So, you added a development dependency to `dotenv` but you're loading in a file that will be executed in client applications that have this gem. Isn't that a problem? If not, why not?

Reply:
Yes, you're right! I removed it!